src/utilities/bot-online-time.ts

```diff
@@ -1,11 +1,8 @@
+import { DateTime } from 'luxon';
+
 export function getBotOnlineAt(): string {
-  const date = new Date();
-  const hour = date.getHours() < 10 ? `0${date.getHours()}` : date.getHours();
-  const mins =
-    date.getMinutes() < 10 ? `0${date.getMinutes()}` : date.getMinutes();
-  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-  const tz = /\((.*)\)/.exec(new Date().toString())![1];
-  const am_pm = hour >= 12 ? 'PM' : 'AM';
+  const dt = DateTime.local();
```

Comment:
Another question, while I'm testing this. 

Using the local format means the time on my instance is different than the times I saw when you were testing your instance. When this is deployed to Heroku, do we know that the local format will be compatible?

Reply:
No, I already asked Brad that it will be local to the server the bot is up at. So depending on where it's at, it will always change the time to the local area of where the bot is, but the format will always be in `[HH]:[MM]:[SS] [AM/PM] [TimeZone]`

Reply:
Here's hoping the bot doesn't end up in the middle of the Atlantic Ocean then 🙃